### PR TITLE
added faq on missing certificates in trust stores

### PR DESCRIPTION
**Pull request by AndreasEhret**

### CODE REVIEW

File: documentation/faq/FAQ.adoc [hunk 1]
```diff
@@ -71,7 +71,20 @@ last modified timestamp of the resource and will go ahead and download the resou
...
+OpenWebStart does not maintain a curated collection of certificates by itself.
+Rather it relies on the JVM which brings a default set of certificates.
+
+In this context it is helpful to distinguish between the bundled JVM, used to run OpenWebStart itself, and the custom-selected JVM used to launch the JNLP applications.
+The second one is determined by the definition in the JNLP file or by the configuration via OpenWebStart JVM Manager.
+
+With its half-yearly releases (spring and fall) we update the bundled JVM and thus the certificates included in the internal JVM.
+
+During the execution of the JNLP application the certificates of the custom-selected JVM are available.
+So the control which certificates will be available depends on the definitions in the JNLP file and the configuration via OpenWebStart JVM Manager.
```
Comment (janakmulani) [inline documentation/faq/FAQ.adoc:87]: ...depends on the JVM definitions...

**Comment by sclassen** [comment]
Like it :+1: 

Could you also add a paragraph on how to add a certificate to the OWS trust store (which will be available in stage 1 and stage 2)

File: documentation/faq/FAQ.adoc (hunk 1 above)
Comment (AndreasEhret) [reply documentation/faq/FAQ.adoc]: improved wording

**Comment by AndreasEhret** [comment]
> Like it 👍
> 
> Could you also add a paragraph on how to add a certificate to the OWS trust store (which will be available in stage 1 and stage 2)

actually never did that. Can it be done by the Settings?

**Pull request merged**